src/applications/simple-forms/shared/tests/e2e/helpers.js

```diff
@@ -81,6 +81,19 @@ export const selectGroupCheckboxWidget = label => {
   }
 };
 
+export const selectCheckboxGroupWebComponent = data => {
+  if (data && typeof data === 'object') {
```

Comment:
just FYI, I've refactored a lot of these patterns in my [Cypress refactor PR](https://github.com/department-of-veterans-affairs/vets-website/pull/26142/files#:~:text=export%20const%20selectCheckboxWebComponent%20%3D%20(fieldName%2C%20condition)%20%3D%3E%20%7B), but this looks good to me. I didn't actually make a group checkbox cypress command because the regular checkbox sort of handles it already. but this is great for addressing the pattern as a whole.